packages/forms-web-app/src/views/before-you-start/local-planning-department.njk

```diff
@@ -0,0 +1,76 @@
+{% extends "layouts/main.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %},
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+
+{% set title="Which local planning department dealt with your planning application? - Before you start - Appeal a householder planning decision - GOV.UK" %}
+
+{% block pageTitle %}
+  {% if errors %}
+    {% set title="Error: " + title %}
+  {% endif %}
+  {{ title }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errorSummary %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-visually-hidden" data-cy="eligible-departments">{{ eligibleDepartments }}</p>
```

Comment:
I may be missing something. But I could not find the use of these hidden fields.

Reply:
Me neither. I was keeping to earlier versions of the page as the goal is to keep them similar, so I'm glad you've flagged this. Removed.